Softeq.XToolkit.WhiteLabel/Dialogs/AlertDialogConfig.cs

```diff
@@ -0,0 +1,33 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.XToolkit.WhiteLabel.Dialogs
+{
+    /// <summary>
+    ///     Presents an alert dialog to the application user with a single cancel button.
+    /// </summary>
+    public class AlertDialogConfig : IDialogConfig<object?>
+    {
+        public AlertDialogConfig(string title, string message, string cancelButtonText = "OK")
```

Comment:
Default values of string won't be really useful in localizable projects, and might even be harmful